packages/terra-alert/src/Alert.module.scss

```diff
@@ -90,7 +90,7 @@
     color: var(--terra-alert-custom-text-color, #1c1f21);
   }
 
-  .actions > :first-child {
+  .actions > :nth-child(2) {
```

Comment:
If you used padding vs margin, could you add the CSS to the action class and not need to tie to a numbered child.  If your layout changes, you are susceptible to a defect being introduced.

Reply:
This change has been undone with the latest changes for review, for context this was updated since we temporarily added an empty span so the action element no longer had the correct CSS